### ISSUE TITLE
Porting Rules Once per Day

### ISSUE THREAD

**Issue by bugfolder** [backdrop-ops/contrib#428]
**Name of the module, theme, or layout you are working on**
Rules Once per Day

**Link to the drupal.org module, theme, or layout**
https://www.drupal.org/project/rules_onceperday

**(Optional) Link to the drupal.org issue where you notified the maintainer(s) of this port**
https://www.drupal.org/project/rules_onceperday/issues/3169326

**Do you need assistance with this port?**
Not yet (but I'll holler if I run into trouble).

# Please return to this issue when complete, and change the label to "Port complete".

**Comment by stpaultim** [comment]
Good luck!

**Comment by bugfolder** [comment]
That was pretty quick. I've got a version that mostly works, but the default rule example doesn't. This seems to be because `hook_default_rules_configuration()` isn't yet properly supported by Rules module, and that seems to be because the core Entity module doesn't yet support the `'export' => array('default hook' => 'default_rules_configuration')`  declaration in `rules_entity_info()`. I've posted an issue in the Rules issue queue.
